algorithms/HPdutility/man/hpdsample.Rd

```diff
@@ -0,0 +1,85 @@
+\name{hpdsample}
+\alias{hpdsample}
+\title{A sampling algorithm for darrays/dframes}
+\description{
+    Samples data from data1 (and when present, data2, maintaining the same
+    order of samples between them) into a new darray/dframe with nSamplePartitions
+    partitions, each of which contains approximately samplingRatio * nrow(data1)
+    elements.
+}
+\usage{
+hpdsample(data1, data2, nSamplePartitions, samplingRatio, trace = FALSE)
+}
+
+\arguments{
+  \item{data1}{
+      A darray/dframe to sample
```

Comment:
It is a common habit in R community that definition of the arguments start with lower case; let's be consistent to the others. Moreover, we should mention that data1 must be row-wise partitioned. So, I suggest:
a darray/dframe to sample. It must be row-wise partitioned.